modules/virt-storage-wizard-fields-web.adoc

```diff
@@ -26,3 +26,36 @@
 |The `StorageClass` that is used to create the disk.
 
 |===
+
+[id="virt-storage-wizard-fields-advanced-web_{context}"]
+[discrete]
+== Advanced storage settings
+The following advanced storage settings are available for *Blank*, *URL*, and *Attach Cloned Disk* disks.
+These parameters are optional. If you do not specify these parameters, the defaults are used, based on what has been configured in the `kubevirt-storage-class-defaults` ConfigMap.
+
+[cols="2a,3a,5a",caption=]
```

Comment:
What is `caption=` for? I am unfamiliar.

Reply:
When you give a table a header, by default it gets a 'Table [0-9]. ' prefix. This voids that prefix.
But I had since added a lvl2 section header and preceding paragraph, which made the table header redundant anyway so... it gone!